src/components/CommentsList.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react';
+import { useAppSelector } from 'hooks/useRedux';
+import { selectComments } from 'store/comments/commentsSelectors';
+import Loader from './Loader';
+import CommentItem from './CommentItem';
+
+const CommentsList:FC = () => {
+  const comments = useAppSelector(selectComments);
+
+  if (!comments) {
+    return <Loader />;
+  }
+
+  if (!comments.length) {
+    return (
+      <p className="title is-4" data-cy="NoCommentsMessage">
+        No comments yet
+      </p>
+    );
+  }
+
+  return (
+    <article style={{ marginBottom: '1.5rem' }}>
```

Comment:
semantic tag for the list will be `ul` or `ol`

the same for item:  ~section~ => `li`